### DISCUSSION TITLE
2 Suggestions for Improvement

### DISCUSSION THREAD

**Discussion by tobylibo** [jesec/flood#308] [General]
Thanks to the developers for providing such an excellent program. Currently the program works perfectly with qbittorrent on the computer, for me. 

However, I found that there are 2 problems when using the browser on the mobile phone. I hope these two problems can be improved in the next version. 1. Add that you can automatically hide the left sidebar by clicking on the download content on the right side, or automatically recognize the mobile device for browsing, and the left sidebar will not be displayed directly after logging in. 2. If you browse in the Expanded View mode, the display is complete when the browser is narrowest, but a part of it cannot be displayed on the iphone (as shown in the figure).

Please forgive my bad English, because I am Chinese. Thanks again to the programmer！

![Xnip2021-05-19_10-22-31](https://user-images.githubusercontent.com/76021750/118747262-422b5380-b88c-11eb-89b4-e6ea87449575.jpg)
![Xnip2021-05-19_10-23-12](https://user-images.githubusercontent.com/76021750/118747265-43f51700-b88c-11eb-98b8-7ccecca04ff5.jpg)

**Comment by jesec** [comment]
Oh. You can just decrease the number of cells. Also the sidebar is already hidden by default when you use a mobile device.

**Reply by tobylibo** [reply]
Thank you for your prompt reply. At present, I can only follow the method you said.
